files/build_templates/docker_image_ctl.j2

```diff
@@ -7,26 +7,29 @@ function getMountPoint()
 
 function getBootType()
 {
-    local TYPE
-    case "$(cat /proc/cmdline)" in
-    *SONIC_BOOT_TYPE=fast*)
-        TYPE='fast'
-        ;;
-    *SONIC_BOOT_TYPE=warm*)
+    local BOOT_TYPE
+    case "$(cat /proc/cmdline | grep -o 'SONIC_BOOT_TYPE=\S*' | cut -d'=' -f2)" in
```

Comment:
>SONIC_BOOT_TYPE= [](start = 41, length = 16)

We should handle both cases for backward-compatible with 201803:
1. fast-reboot
2. SONIC_BOOT_TYPE=fast-reboot

Otherwise we cannot fast-reboot from 201803 into 201811.